lock.yml

```diff
@@ -0,0 +1,14 @@
+# Locking closed, inactive issues and pull requests
+name: Locking issues and pull requests
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: fastlane/github-actions/lock@latest
+      with:
+        repo-token: ${{ secrets.BOT_GITHUB_TOKEN }}
+        days-before-lock: 60
```

Comment:
The time before locking is taken from an original bot's implementation:

https://github.com/fastlane/issue-bot/blob/457348717d99e5ffde34ca1619e7253ed51ec172/bot.rb#L286-L292